lib/matrack/utility.rb

```diff
@@ -9,4 +9,20 @@ def to_camel_case
     return self if self !~ /_/ && self =~ /[A-Z]+.*/
     split("_").map(&:capitalize).join
   end
+
+  def to_b
+    return true if !self.nil? && (self == true || downcase == "true")
+    return false if self.nil? || downcase == "false" || self == false
+    false
+  end
+
+  def titleize
+    split.map!(&:capitalize).join(" ")
+  end
+end
+
+class ::Array
+  def hash_getter
+    first.select{|k| k.is_a? String}
```

Comment:
Space missing to the left of {.<br>Space between { and | missing.<br>Space missing inside }.